lib/rohd_hcl.dart

```diff
@@ -12,20 +12,22 @@ export 'src/encodings/encodings.dart';
 export 'src/error_checking/error_checking.dart';
 export 'src/exceptions.dart';
 export 'src/extrema.dart';
-export 'src/fifo.dart';
 export 'src/find.dart';
 export 'src/find_pattern.dart';
 export 'src/gaskets/spi/spi_gaskets.dart';
 export 'src/interfaces/interfaces.dart';
+export 'src/memory/cam.dart';
```

Comment:
you could make a `src/memory/memory.dart` that exports all the memory-related stuff